lib/lib.js

```diff
@@ -82,4 +82,7 @@ const taskOfSnippets = (dependencies, timeout, glob) =>
     .map(List)
     .chain(traverseFiles(timeout))
 
-module.exports = { taskOfSnippets }
+module.exports = {
+  injectDependencies,
```

Comment:
I don't know how you feel about exposing things that "should" be private to be able to test them. Personally, I prefer testability over encapsulation.

Reply:
I'm ok with exposing too, it could even be in other file `helpers` or something like that

Reply:
For now is ok to keep it here, let's first think a bit about code structure/architecture and then move things around